osmaxx/conversion/management/commands/result_harvester.py

```diff
@@ -41,6 +41,10 @@ def _handle_running_jobs(self):
             self._update_job(job_id=job_id, queue=queue)
 
     def _update_job(self, job_id, queue):
+        if job_id is None:
+            logger.exception("job_id is None, None is not a valid id!")
+            return
```

Comment:
yeah, `logger.error` make more sense here - it should only show in the logs as exception/error, not actually throw one.